lib/fizzy/types.hpp

```diff
@@ -15,7 +15,7 @@
 namespace fizzy
 {
 // https://webassembly.github.io/spec/core/binary/types.html#binary-valtype
-enum class ValType : uint8_t
```

Comment:
Why is this needed?

Reply:
To be able to cast between arrays of `fizzy::ValType` and arrays of `FizzyValueType`

Reply:
I don't think we can go with this change.

Reply:
Can you try using `uint8_t` on C side for now, and add
```
const uint8_t FizzyI32 = 0x7f;
const uint8_t FizzyI64 = 0x7e;
...
```

Reply:
Is performance considerations the reason why not?